add email on Media Consent Forms

Adimote opened the pull request.

Adimote committed:
add media consent form email

PeterJCLaw commented on SB2018/2018-03-23.md:
```diff
@@ -0,0 +1,15 @@
+
+Hi all,
+
+In order for anyone to attend the competition, a Media Consent Form must be filled out. If students are over 18, they can fill out the forms themselves, however if students are under 18, it is a legal requirement that a parent/guardian signs the form for them.
```
Minor: a semicolon before "however" in the second sentence would clarify the sentence.

Adimote replied:
fixed in 773f1d347c1f294d953ee7df3e013fecf7113bc0

PeterJCLaw commented on SB2018/2018-03-23.md:
```diff
@@ -0,0 +1,15 @@
+
+Hi all,
+
+In order for anyone to attend the competition, a Media Consent Form must be filled out. If students are over 18, they can fill out the forms themselves, however if students are under 18, it is a legal requirement that a parent/guardian signs the form for them.
```
> if students are under 18, it is a legal requirement that a parent/guardian signs the form for them.

This implies that it is a legal requirement that the form be signed, which isn't what you mean. What you mean is that the form may only be signed by the parent/guardian, rather than the young person.

I'm also not entirely sure how anyone would ever enforce that; certainly it's not a general case that signatures from minors don't count (see: young person' bank accounts).

I'm not sure how best to phrase this, though maybe just dropping the "it is a legal requirement that" part would work; with a little rewording that would give:
> if students are under 18, a parent/guardian must sign the form on their behalf.

Adimote replied:
fixed in 20aae3da391634046833ebfa6d57a9ac2a41ec75

PeterJCLaw commented on SB2018/2018-03-23.md:
```diff
@@ -0,0 +1,15 @@
+
+These forms must be filled and sent to us to get a ticket for the competition, and to ultimately gain entry.
+
+To submit a ticket just send me an email with an attached PDF of the tickets! We may reply with any errors we see.
```
I don't think you mean "ticket" either on this line or the following one.

Adimote replied:
a5b9e06f65f4fc34d860e21a42230aedad76ee79

Adimote replied:
fixed in 2363f601e21f5645280e98d5437df0a7d4308a93

PeterJCLaw reviewed (dismissed):
See inline comments.

Meta question (please direct me to the existing discussion if possible or lets go to a slack/list thread; this PR isn't the right venue for a discussion about this): what's the plan for processing and storing the forms and for generating the tickets? Note that the SR framework for this was very much a bodge and highly coupled to user accounts in LDAP.

Adimote committed:
Change minimum age from 18 to 16

Adimote committed:
fix grammar

Adimote committed:
we aren't asking them to fill out tickets

Adimote committed:
reword parent/guardian

Adimote committed:
add comma

PeterJCLaw commented on SB2018/2018-03-23.md:
```diff
@@ -0,0 +1,14 @@
+
+In order for anyone to attend the competition, a Media Consent Form must be filled out. If students are over 16, they can fill out the forms themselves; however, if students are under 16, the form must be signed by their parent/guardian.
+
+These forms must be filled and sent to us to get a ticket for the competition, and to ultimately gain entry.
```
Maybe set an explicit deadline here?

Adimote replied:
fixed in 418d0e1f8b61c6eae68603410d8d44ae9e473615

PeterJCLaw commented on SB2018/2018-03-23.md:
```diff
@@ -0,0 +1,14 @@
+
+These forms must be filled and sent to us to get a ticket for the competition, and to ultimately gain entry.
+
+To submit a form, just send me an email with an attached PDF of a scan of the the filled out forms. We may reply with any errors we see.
```
Will they be able to bring/fill them out on the day?
If not, what happens to guests?

Adimote replied:
also fixed in 418d0e1f8b61c6eae68603410d8d44ae9e473615

Adimote committed:
mention signing forms at the door and add deadline

- Mention guests can sign forms on the door
- Add a deadline of April 17th for getting the forms in so we can
process them.

Adimote committed:
people must be over 16 to sign on the door

anton-firmware commented on SB2018/2018-03-23.md:
```diff
@@ -0,0 +1,14 @@
+
+To submit a form, just send me an email with an attached PDF of a scan of the the filled out forms. We may reply with any errors we see.
+
+Please try to get the tickets to us as soon as possible, as we'll be extremely busy organising the competition itself in the week before. We will be processing forms up to the 17th of April, if forms aren't filled out by then, you can bring them to the competition to be checked on the door. Any additional guests can sign a form on the door as long as they're 16 or over.
```
> Please try to get the tickets to us as soon as possible, as we'll be e

Should this be MCF not tickets?

Adimote replied:
I could've sworn I fixed that... fixed in f397374

Adimote committed:
fix another mention of tickets instead of forms

anton-firmware reviewed (dismissed):
the latest one lgtm :)

RealOrangeOne commented:
After discussion in #general, we're no longer doing tickets. MCFs should be bought to the door. We'll have additional ones there for extra people / correcting invalid ones.

Commit:
reword MCF email to bring them to the comptition

Adimote committed:
move date

PeterJCLaw commented on SB2018/2018-04-03.md:
```diff
@@ -0,0 +1,13 @@
+In order for anyone to attend the competition, a 'Media Consent' Form must be filled out. If students are over 16, they can fill out the forms themselves; however, if students are under 16, the form must be signed by their parent/guardian.
+
+These forms must be filled out by all attendees and brought to the competition in order to gain entry.
+Any additional guests can sign a form on the door as long as they're 16 or over.
```
Perhaps phrase this to cope with under-16 guests too?
> We'll have some forms on the door for additional guests, though we encourage that everyone completes them ahead of time.

PeterJCLaw replied:
Hrm, given my proposed changed to the following paragraph it might make sense to move this line below that?

I'm not sure where that leaves the line "These forms must be..." though.

Adimote replied:
fixed in 8f9e501dc2fd51aeb106b0311df66208e53ffa1f

PeterJCLaw commented on SB2018/2018-04-03.md:
```diff
@@ -0,0 +1,13 @@
+These forms must be filled out by all attendees and brought to the competition in order to gain entry.
+Any additional guests can sign a form on the door as long as they're 16 or over.
+
+It's important that the form is correctly filled out to ensure the line isn't held up at reception on the Saturday morning.
```
Perhaps phrase this in a manner which is clearer that we'd like the teachers to actually do something?
> In order to ensure we can process arrivals smoothly, please ensure that your team has all completed Media Consent Forms before you arrive at the competition. If you are brining any under 16s, please ensure that their parents or guardians have completed the proper sections as they may be denied entry if the forms are incorrect.

Adimote replied:
fixed in 8f9e501dc2fd51aeb106b0311df66208e53ffa1f

PeterJCLaw reviewed (commented):
Some thoughts

Adimote committed:
We will have forms available for guests

Also rephrase the section to put a requirement on teachers to
check forms are signed correctly.

Adimote committed:
Mention its the same form for extra guests

Remove the ambiguity of 'extra forms' by changing it to
'extra copies of the form'

Adimote committed:
fix spelling error

RealOrangeOne commented:
There's a lack of "you can find the forms attached" or "you can download them here" in this email? Have we decided how we're distributing them?

Adimote committed:
mention the form is attached

Adimote commented:
@RealOrangeOne fixed in 67131af

Adimote commented:
please someone approve this so it can be shipped tomorrow, it's already taken 2 weeks to get this out

anton-firmware reviewed (approved):
lgtm

Adimote committed:
add form to attach

this is the generated output of the media-consent repo

Adimote committed:
update email send dates
